app/views/layouts/angular/_generic_form_buttons.html.haml

```diff
@@ -0,0 +1,27 @@
+.clearfix
```

Comment:
Don't we already have a partial exactly like this? (Meaning same buttons) Why introduce a new one?